client/my-sites/email/email-management/home/email-plan-warning-notice.tsx

```diff
@@ -20,6 +21,12 @@ export const EmailPlanWarningNotice = ( props: EmailPlanWarningNoticeProps ) =>
 	const cannotAddEmailWarningMessage = cannotAddEmailWarningReason?.message ?? '';
 	const cannotAddEmailWarningCode = cannotAddEmailWarningReason?.code ?? null;
 
+	// If email and domain are owned by different users, none of the users will be able to make a purchase and the only way to resolve this
+	// is to reach out to support. Therefore, we should surface a different message to address this scenario.
+	if ( ! canDomainOwnerAddEmail( domain ) ) {
+		return <EmailDifferentDomainOwnerMessage source="email-management" />;
```

Comment:
I'm not sure to understand why we need to specify a `source` parameter: as far I can see we only use `EmailDifferentDomainOwnerMessage` here so we could just hardcode that parameter in that component.

Reply:
You're right, we can simply hardcode this into the `<EmailDifferentDomainOwnerMessage>` component!

Reply:
Done